src/pages-and-resources/discussions/data/thunks.js

```diff
@@ -53,36 +62,19 @@ export function saveAppConfig(courseId, appId, drafts, successPath) {
     dispatch(updateSaveStatus({ status: SAVING }));
 
     try {
-      const {
-        apps,
-        features,
-        activeAppId,
-        appConfig,
-        discussionTopicIds,
-        discussionTopics,
-        divideDiscussionIds,
-      } = await postAppConfig(courseId, appId, drafts);
+      const apps = await postAppConfig(courseId, appId, drafts);
+      dispatch(updateAppState(apps));
 
-      dispatch(addModel({ modelType: 'appConfigs', model: appConfig }));
-      dispatch(addModels({ modelType: 'apps', models: apps }));
-      dispatch(addModels({ modelType: 'features', models: features }));
-      dispatch(addModels({ modelType: 'discussionTopics', models: discussionTopics }));
-
-      dispatch(loadApps({
-        activeAppId,
-        appIds: apps.map(app => app.id),
-        featureIds: features.map(feature => feature.id),
-        discussionTopicIds,
-        divideDiscussionIds,
-      }));
       dispatch(updateSaveStatus({ status: SAVED }));
       // Note that we redirect here to avoid having to work with the promise over in AppConfigForm.
       history.push(successPath);
     } catch (error) {
       if (error.response && error.response.status === 403) {
-        dispatch(updateSaveStatus({ status: DENIED }));
-        // This second one will remove the interface as well and hide it from the user.
-        dispatch(updateStatus({ status: DENIED }));
+        const newStatus = { status: DENIED }
+        if (error.response.data.detail) {
+          newStatus.messages = [error.response.data.detail];
+        }
```

Comment:
I don't think we should use the backend response directly here, instead have a friendlier mesage.

Reply:
The friendlier message will have to be generic, then, because we don't know why we got a 403 response.